app/modules/win32ipc/module.cc

```diff
@@ -6,17 +6,24 @@
 
 #include "pipe_wrapper.h"
 
+#define NORMAL_STRING v8::String::NewStringType::kNormalString
+v8::Local<v8::String> newFromUtf8(v8::Isolate* isolate, char const* str) {
+        v8::MaybeLocal<v8::String> v = v8::String::NewFromUtf8(isolate, str,
+                v8::String::NewStringType::kNormalString);
+        return v.ToLocalChecked();
+}
+
 void CreatePipe(v8::FunctionCallbackInfo<v8::Value> const& args) {
     auto isolate = v8::Isolate::GetCurrent();
 
     if (args.Length() != 1) {
         isolate->ThrowException(v8::Exception::TypeError(
-            v8::String::NewFromUtf8(isolate, "Wrong number of arguments")));
+            v8::String::NewFromUtf8(isolate, "Wrong number of arguments", NORMAL_STRING)));
```

Comment:
you should use the function declared above instead for all of these. i'm guessing that you probably see build warnings because this ends up calling the deprecated function still.